src/Helpers/MultiFieldsEntityTrait.php

```diff
@@ -34,4 +36,52 @@ public function toArray(): array
             return ($value !== null);
         });
     }
+
+    public function filterAdditionalProperties($additionalProperties): void
+    {
+        foreach ($additionalProperties as $key => $value) {
+            $key = $this->dashesToCamelCase($key, true);
+            //Check if $additionalProperties is already an entities
+            $path_to_properties = \GingerPluginSdk\Client::PROPERTIES_PATH . $key;
+            if (class_exists($path_to_properties)) {
+                if ($value instanceof $path_to_properties) {
+                    $this->$key = $value;
+                } else {
+                    $this->$key = new $path_to_properties($value);
+                }
+                continue;
+            }
```

Comment:
Checking of `PROPERTIES_PATH `  on lines 44-53 looks very same as checking `ENTITIES_PATH`  which is a little lower in the code on lines 70-78. Is it possible move that code in some function? 

I think about something like that:
```
  public function checkPathes($path_for_check)
  {
           if (class_exists($path_for_check)) {
                if ($value instanceof $path_for_check) {
                    $this->$key = $value;
                } else {
                    $this->$key = new $path_for_check(...$value);
                }
                continue;
            }
  }
```
But it could be unnecessary complication. What do you think about it?

Reply:
I think it's was a good notice that the body of if statement is pretty similar in this cases. But if i will move this body to a function, it's looks that only in these places it will be used. So, for me, such a method is not pays.